tests/functional/lib/helpers.js

```diff
@@ -975,30 +979,31 @@ define([
     };
   }
 
-  function openPage(context, url, readySelector) {
-    var remote = getRemote(context);
-    return remote
-      .get(require.toUrl(url))
-      .setFindTimeout(config.pageLoadTimeout)
+  function openPage(url, readySelector) {
+    return function () {
```

Comment:
It could, in a slightly different way. I'd have to do something like:

```js
var openPage = thenify(function (url, readySelector) {
...
```

That's totally reasonable, just different to how it's done elsewhere. Basically, I think what you suggest is the Right Thing To Do, but I've been waffling.